crates/rune/src/runtime/from_value.rs

```diff
@@ -120,7 +120,7 @@ pub trait FromValue: 'static + Sized {
 /// calls.
 pub trait UnsafeFromValue: Sized {
     /// The output type from the unsafe coercion.
-    type Output: 'static;
+    type Output;
```

Comment:
This has the same issue pointed out before, but it's at least highlighted a bit more by having to use `unsafe_from_value`. And of course that function could be marked as actually `unsafe`.

Haven't dug into it yet but I'm also curious if this trait could be automatically derived, and therefore made private.